src/transforms/transform-pull-request.ts

```diff
@@ -52,66 +52,33 @@ export const transformPullRequest = async (gitHubInstallationClient: GitHubInsta
 	const prBody = await booleanFlag(BooleanFlags.ASSOCIATE_PR_TO_ISSUES_IN_BODY, true) ? body : "";
 	const issueKeys = jiraIssueKeyParser(`${prTitle}\n${head.ref}\n${prBody}}`);
 
-	const logPayload = {
-		prTitle: prTitle || "none",
-		repoName: head?.repo?.name || "none",
-		prRef: pullRequest.head.ref || "none"
-	};
-
 	if (isEmpty(issueKeys) || !head?.repo) {
-		log?.info(logPayload, "Ignoring pullrequest since it has no issue keys or repo");
+		log?.info({
+			pullRequestBranchName: pullRequest.head.ref,
```

Comment:
Will it be hashed?

Reply:
ha, did not even register that it was UGC. no no it will not
I shall drop it